docs/src/tutorial/cython_tutorial.rst

```diff
@@ -255,49 +326,57 @@ sense for those lines to be yellow. Same for the list comprehension because
 it involves the creation of a Python object. But the line ``if n % i == 0:``, why?
 We can examine the generated C code to understand:
 
-.. figure:: python_division.png
+.. tabs::
+    .. group-tab:: Cython
+
+        .. figure:: python_division.png
+
+    .. group-tab:: Pure Python
+
+        .. figure:: python_division_pure.png
```

Comment:
```suggestion
.. tabs::
    .. group-tab:: Cython

        .. figure:: python_division_pyx.png

    .. group-tab:: Pure Python

        .. figure:: python_division_py.png
```

Reply:
accepted, at the same time changing [this](https://github.com/cython/cython/pull/4226/commits/cc3ab8163ff55dd1d8ac0e8e76063a76b0f24f9c) and [that](https://github.com/cython/cython/pull/4226/commits/c3cda8b03606ae58023f1134f3be221658b2076e)